compiler/damlc/tests/daml-test-files/Interface.daml

```diff
@@ -1,7 +1,7 @@
 -- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
 
--- @SINCE-LF-FEATURE DAML_INTERFACE
+-- @SINCE-LF-FEATURE DAML_INTERFACE_EXTENDED
```

Comment:
This test should still work with basic interfaces, otherwise we wouldn't be testing 1.15. This is probably also the case for a few other `daml-test-file`s

```suggestion
-- @SINCE-LF-FEATURE DAML_INTERFACE
```